rust-connectors/sources/http/src/bin/main.rs

```diff
@@ -59,9 +38,10 @@ async fn main() -> Result<()> {
 
     tracing::info!("Initializing HTTP connector");
     tracing::info!(
```

Comment:
Could you change this to:

```rust
tracing::info!(
   interval = %opts.interval,
   method = %opts.interval,
   topic = %opts.common.fluvio_topic,
   output_format = %opts.output_format,
   endpoint = %opts.endpoint
  )
 ```

Reply:
:+1: Done - I think opts.method tho

Reply:
btw - is it possible to toggle off the non-monotype font? My eyes / brains are too used to monotype.